e2eTests/src/test/java/IngestionTest.java

```diff
@@ -225,7 +234,7 @@ private void triggerAccessionMessageFromCEGA()
     String message =
         String.format(
             "{\"type\":\"accession\",\"user\":\"%s\",\"filepath\":\"/p11-dummy@elixir-europe.org/files/%s\",\"accession_id\":\"%s\", \"decrypted_checksums\": [ { \"type\":\"sha256\", \"value\": \"%s\" },{\"type\":\"md5\", \"value\": \"%s\"} ] }",
```

Comment:
Probably best to format all JSON strings using formatter. Avoids accidental errors.

Reply:
Addressed in [2a59f2a](https://github.com/ELIXIR-NO/FEGA-Norway/pull/124/commits/2a59f2af44be312d5c2495b2cf17e58fd5b77791)